mne/io/pick.py

```diff
@@ -928,7 +944,8 @@ def _pick_aux_channels(info, exclude='bads'):
 def _pick_data_or_ica(info, exclude=()):
     """Pick only data or ICA channels."""
     if any(ch_name.startswith('ICA') for ch_name in info['ch_names']):
-        picks = pick_types(info, exclude=exclude, misc=True)
+        # FIXME: is meg=True really correct here?
+        picks = pick_types(info, exclude=exclude, misc=True, meg=True)
```

Comment:
@larsoner I don't really understand why `meg=True` here - shouldn't this pick only ICA channels (i.e. `misc` channels)?

Reply:
It's probably not correct. If you want to be paranoid you could do it both ways (make a `picks2`), `np.testing.assert_array_equal` the two, and run all `preprocessing` tests to double check, then remove the `meg=True` one knowing it never mattered. But it shouldn't, and if it does, it's probably a bug...

Reply:
But we could do that in a separate issue so that this PR replicates the previous behavior? Or should I correct it (provided that `picks` equals `picks2` in the tests)?

Reply:
Either way is fine by me

Reply:
OK, then I'd prefer tackling this in a new issue/PR.